libs/order/src/reducers/order-store-feature-key.ts

```diff
@@ -0,0 +1 @@
+export const DAFF_ORDER_STORE_FEATURE_KEY = 'daffOrder';
```

Comment:
what is the purpose of this key?

Reply:
So that the key that the state is stored in doesn't have to be hardcoded, kinda the same rationale for using an enum instead of just hardcoded strings. It will be used in the feature state selector.